vespajlib/src/main/java/com/yahoo/time/UncheckedTimeoutException.java

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.time;
+
+/**
+ * Exception thrown when a blocking operation times out.
+ *
+ * <p>Mirrors {@link java.util.concurrent.TimeoutException}
+ *
+ * @author hakon
+ */
+@SuppressWarnings("serial")
+public class UncheckedTimeoutException extends RuntimeException {
```

Comment:
I suggest removing the "Unchecked" prefix. It is used when there is already a checked variant in the same package, but there is not here.

Reply:
There's also `com.google.common.util.concurrent.UncheckedTimeoutException` from Guava, which is used in several places already.

Reply:
OK, done.

Reply:
For some reason github didn't show Martin's comment before now. Moving to guava.